modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -142,6 +147,9 @@ search-results-display-context-help=Access the data contained in the SearchResul
 search-results-portlet-instance-configuration-name=Search Results Configuration
 search-results-template-key-default=Search Results Template Key Default
 search-results-web-template-configuration-name=Search Results Web Template Configuration Name
+category-facet-display-context=Category Facet Display Context
+category-facet-display-context-help=Access helper methods to retrieve the Category Facet widget configuration.
```

Comment:
@oliv-yu Following Russ's recent comment on https://github.com/thektan/liferay-portal/pull/119#discussion_r365403009, can we also update this?

> Access the data contained in the AssetCategoriesSearchFacetDisplayContext object.

Reply:
@oliv-yu Actually I'm going to make some changes to the FTL layout, so I can make this change too.

Reply:
Okay! Let me know if I need to take any action